code/modules/forensics/atom_forensic.dm

```diff
@@ -201,6 +199,9 @@
 				#else
 				CI.UpdateOverlays(null, "blood_splatter")
 				#endif
+				if (istype(src, /obj/item/clothing))
+					var/obj/item/clothing/C = src
+					C.clean_stains()
```

Comment:
does this need to be at this indentation level?

Reply:
Moved up.